### DISCUSSION TITLE
Import Sprite Sheets

### DISCUSSION THREAD

**Discussion by WimpyKidFan2007** [Techokami/SonicWorldsNext#128] [Q&A]
How do I import sprite sheets from the spriters resource and deviantart into Sonic Worlds Next?

**Comment by Techokami** [comment]
Oh my God, did you not even bother to answer the question I asked you in your previous thread? Please go back and answer that question now.

**Reply by WimpyKidFan2007** [reply]
Okay I answered the question in the previous thread.

**Comment by DimensionWarped** [comment]
I'm going to assume you are talking about character sprites since that's always what a newbie wants to do... but I'm just going to tell you that replacing a character sprite isn't for the faint of heart. You'll basically need to manually place every frame of the sprite over top of the the original in an image editor... and if the number of frames doesn't match up to the original the effort involved is going to be much higher. At that point you would have to start making modifications to the character's sprite object and animation player to make things fit. We're talking like up to dozens of hours of continually breaking and fixing your animations, and that's assuming you start from a position of knowing what you are doing (and you don't!).

I'd recommend you start off just by making some enemy sprites or something. They tend to be a lot simpler. You need to learn the basics before you start trying to tackle big tasks.
